vignettes/creating-custom-modules.Rmd

```diff
@@ -15,170 +15,230 @@ vignette: >
 The `teal` framework provides a large catalog of plug-in-ready analysis modules to be incorporated into `teal` applications.
 However, it is also possible to create your own modules using the `module` function.
 
-## Components of a module
+In this guide, we will convert a simple histogram formula into a robust `teal` module:
+```r
+hist(dataset[, selected], las = 1)
```

Comment:
Good idea, but maybe avoid using the `input_` naming convention. The goal is to replace `dataset` with `input_dataset` when writing this in the server logic.

Reply:
Update was done in c508c133cbc1c128a9abbf530b15ec42cff8d338

Reply:
I was checking this and realized the variables names under the section "Understanding the Inputs and Requirements" doesn't match these (it has `selected` instead of `vars` or `variable`).

Reply:
nice catch!

Reply:
Done in 5617508983db5686177b3260a469fdbb71a0e6fb